third_party/open-policy-agent/gatekeeper/helmify/kustomization.yaml

```diff
@@ -0,0 +1,22 @@
+namespace: "{{ .Release.Namespace }}"
+commonLabels:
+  app: '{{ template "eraser.name" . }}'
+  chart: '{{ template "eraser.name" . }}'
+  heritage: '{{ .Release.Service }}'
```

Comment:
we should rename these as these were meant for helm v2
see best practice labels here: https://github.com/open-policy-agent/gatekeeper/issues/1498